numpy/ma/extras.py

```diff
@@ -257,7 +258,8 @@ def getdoc(self):
         npfunc = getattr(np, self.__name__, None)
         doc = getattr(npfunc, '__doc__', None)
         if doc:
-            sig = self.__name__ + ma.get_object_signature(npfunc)
+            sig = self.__name__ + ma.get_object_signature(npfunc) + '\n'
+            doc = inspect.cleandoc(doc)
             locdoc = "Notes\n-----\nThe function is applied to both the _data"\
                      " and the _mask, if any."
```

Comment:
```suggestion
            locdoc = ("Notes\n-----\nThe function is applied to both the _data"
                      " and the _mask, if any.")
```
Might as well take the opportunity to clean up the style here a bit as well.